src/staging/src/templates/site.js

```diff
@@ -0,0 +1,500 @@
+import React, { useEffect, useState, useRef } from "react"
+import axios from "axios"
+import { sortBy } from "lodash"
+import { graphql } from "gatsby"
+import { Helmet } from "react-helmet"
+import {
+  Box,
+  Container,
+  Grid,
+  CssBaseline,
+  Link,
+  Breadcrumbs,
+} from "@material-ui/core"
+import { makeStyles } from "@material-ui/core/styles"
+import useScrollSpy from "react-use-scrollspy"
+
+import Footer from "../components/footer"
+import Header from "../components/header"
+import PageMenu from "../components/pageContent/pageMenu"
+
+import AccessibilityDetails from "../components/park/accessibilityDetails"
+import AdvisoryDetails from "../components/park/advisoryDetails"
+import Heading from "../components/park/heading.js"
+import ParkActivity from "../components/park/parkActivity"
+import ParkFacility from "../components/park/parkFacility"
+import ParkHeader from "../components/park/parkHeader"
+import ParkOverview from "../components/park/parkOverview"
+import ParkPhotoGallery from "../components/park/parkPhotoGallery"
+import MapLocation from "../components/park/mapLocation"
+import ScrollToTop from "../components/scrollToTop"
+
+import "../styles/parks.scss"
+
+const drawerWidth = 230
```

Comment:
Since this variable is no longer scoped to a single template, it should probably be abstracted further back. Not a big deal since we'll be re-writing this template in the not-too-distant future, but good practice.